app/assets/javascripts/components/generic_object/generic-object-definition-toolbar.js

```diff
@@ -70,8 +70,8 @@ function genericObjectDefinitionToolbarController(API, miqService, $window) {
       .catch(miqService.handleFailure);
   }
 
-  function postAction(response) {
-    var entityName = response.name;
+  function postAction() {
+    var entityName = toolbar.entityName;
     var saveMsg = sprintf(__('%s:"%s" was successfully deleted'), toolbar.entity, entityName);
```

Comment:
@ZitaNemeckova please, add a space behind the semicolon to make this typographically correct:
`'%s: "%s" was successfully deleted'`

Reply:
Done :)